### PR TITLE
👔 Add newsPress tag to iscnPayload

### PR DESCRIPTION
**Pull request by AuroraHuang22**

### CODE REVIEW

**Review by nwingt — APPROVED** [review]

File: pages/nft/url/index.vue [hunk 1]
```diff
@@ -287,8 +286,11 @@ export default class FetchIndex extends Vue {
       publisher,
       type = 'CreativeWork',
     } = this.iscnData
-    let { description = '' } = this.iscnData
+    let { description = '', keywords = '' } = this.iscnData
```
Comment (williamchong) [inline pages/nft/url/index.vue:289]: Will this duplicate with https://github.com/likecoin/app-like-co/pull/486/files?
Should we check if `NewsPress` already exists?

Comment (AuroraHuang22) [reply pages/nft/url/index.vue:289]: [🐛 Prevent duplicate keywords](https://github.com/likecoin/app-like-co/pull/488/commits/f5ac429246948669482dbcf222c48ac9ef597e74)

File: pages/nft/url/index.vue [hunk 2]
```diff
@@ -287,8 +286,11 @@ export default class FetchIndex extends Vue {
...
     description = this.truncate(description, 200)
+    if (!keywords.includes('NewsPress')) {
```
Comment (williamchong) [inline pages/nft/url/index.vue:291]:
```suggestion
    if (this.isNewsPress && !keywords.includes('NewsPress')) {
```

We still need to check this.isNewsPress?

Comment (AuroraHuang22) [reply pages/nft/url/index.vue]: [🐛 Prevent duplicate keywords](https://github.com/likecoin/app-like-co/pull/488/commits/f4a88038960325042a9c758303d5158235a339e7)

**Review by williamchong — APPROVED** [review]

**Pull request merged**